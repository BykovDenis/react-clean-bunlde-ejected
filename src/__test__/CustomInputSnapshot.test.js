import React from 'react';
import CustomInput from '../components/custom-input/index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const value = 'once';
  const tree = renderer
    .create(<CustomInput value={value} changeInput={() => console.log('11')} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});