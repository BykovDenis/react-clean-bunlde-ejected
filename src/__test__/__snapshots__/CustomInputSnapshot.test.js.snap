// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiInput-root-1 MuiInput-underline-5 custom-input"
>
  <input
    aria-invalid={undefined}
    autoComplete={undefined}
    autoFocus={undefined}
    className="MuiInput-input-9"
    defaultValue="once"
    disabled={undefined}
    id={undefined}
    name={undefined}
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    onKeyDown={undefined}
    onKeyUp={undefined}
    placeholder={undefined}
    readOnly={undefined}
    required={undefined}
    rows={undefined}
    type="text"
    value="once"
  />
</div>
`;
